nucliadb/nucliadb/one/tests/test_field_external_file.py

```diff
@@ -28,7 +28,9 @@
 TEST_EXTERNAL_FILE_FIELD_PAYLOAD = {
     "file": {
         "uri": EXTERNAL_FILE_URL,
+        "content_type": "video/webm",
```

Comment:
Maybe you should change the extension of `EXTERNAL_FILE_URL` or create a second payload

Reply:
It doesn't matter much, as it is just test data. 

However, I set it to be different than pdf explicitly, so that is clear that whatever the user inputs will be stored.